tree/dataframe/inc/ROOT/RDF/RMergeableValue.hxx

```diff
@@ -132,7 +140,7 @@ class RMergeableValue : public RMergeableValueBase {
    friend std::unique_ptr<RMergeableValue<T1>> MergeValues(std::unique_ptr<RMergeableValue<T1>> OutputMergeable,
                                                            std::unique_ptr<RMergeableValue<Ts>>... InputMergeables);
    template <typename T1, typename... Ts>
-   friend void MergeValues(RMergeableValue<T1> &OutputMergeable, const RMergeableValue<Ts> &... InputMergeables);
+   friend void MergeValues(RMergeableValue<T1> &OutputMergeable, const RMergeableValue<Ts> &...InputMergeables);
```

Comment:
So far we never neeeded it, but at the first PPP where I presented the design of RMergeableValue and friends variadicity was established as a requirement after discussion with the audience. In the future, if we ever decide to do a reduction phase in distrdf in chunks, this will come in handy